src/pages/MainPage/MainPage.jsx

```diff
@@ -1,21 +1,52 @@
-import React from "react";
-import "../../App.css";
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import "../MainPage/MainPage.css";
 import { useDate } from "../../hooks/useDate";
 
 const MainPage = () => {
+  const [quote, setQuote] = useState({ content: "", author: "" });
```

Comment:
Can store the parameters inside useState as a separate variable named as defaultQuoteData.